src/Raven.Server/Documents/Indexes/MapIndexBase.cs

```diff
@@ -122,15 +122,15 @@ private int UpdateIndexEntriesCorax(IndexItem indexItem, IEnumerable mapResults,
 
         private int UpdateIndexEntriesLucene(IndexItem indexItem, IEnumerable mapResults, Lazy<IndexWriteOperationBase> writer, TransactionOperationContext indexContext, IndexingStatsScope stats)
         {
-            bool mustDelete;
-            using (_stats.BloomStats.Start())
+            if (indexItem.SkipLuceneDelete == false)
             {
-                mustDelete = _filters.Add(indexItem.LowerId) == false;
+                using (_stats.BloomStats.Start())
```

Comment:
Please note that now you're including the duration of  `writer.Value.Delete(indexItem.LowerId, stats);` as bloom filter processing

Reply:
Correct. Fixed.